app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    mv, file, path_to = command.split()
+
+    if mv == "mv" and len(command.split()) == 3 and file != path_to:
```

Comment:
Don't use `split` twice. You can create varable for storring result of `split` and use it where you need.